### PR TITLE
Fix for weather.com.cn

### PR DESCRIPTION
**Pull request by felixonmars**
- Remove background
- Invert logo and chart

Example urls:
http://www.weather.com.cn/weather1d/101200101.shtml
http://www.weather.com.cn/weather40d/101200101.shtml
http://www.weather.com.cn/weather1dn/101200101.shtml

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5762,6 +5762,19 @@ INVERT
...
 
+weather.com.cn
+
+INVERT
+.w_logo
+#hourHolder
+
+CSS
+.lv1, .lv2, .weatherBg01, .weatherBgAll, .weatherBgAll01, .weatherBgAll02, .weatherBgAll03 {
+    background: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5773]: Background to background-image

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5762,6 +5762,19 @@ INVERT
...
 ================================
 
+weather.com.cn
+
+INVERT
+.w_logo
+#hourHolder
+
+CSS
+.lv1, .lv2, .weatherBg01, .weatherBgAll, .weatherBgAll01, .weatherBgAll02, .weatherBgAll03 {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5772]: A new line for each Selector.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config]: Updated.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config]: Updated.

**Pull request merged**